src/utils.js

```diff
@@ -82,7 +82,7 @@ export function saferEvalNoReturn(expression, dataContext, additionalHelperVaria
     if (Object.keys(dataContext).includes(expression)) {
         const exec = expressionCache[codeRet] || sandbox.compile(codeRet)
         expressionCache[codeRet] = exec
-        const methodReference = exec(dataContext, additionalHelperVariables)
+        const methodReference = exec(window, dataContext, additionalHelperVariables)
```

Comment:
this bunch of stuff doesn't probably need the sandbox since it runs only if the expression matches a property of the datacontext so it can probably become
const methodReference = dataContext[expression]

does it make sense to you? I know it was already in the original implementation but I think we can improve it a bit here.

Reply:
yeah sounds good